roles/config-timezone/README.md

```diff
@@ -0,0 +1,19 @@
+set-timezone role
```

Comment:
These files have already been delivered in the previous PR - please eliminate from here